tgui/packages/tgui/interfaces/Fabrication/MaterialAccessBar.tsx

```diff
@@ -157,7 +157,7 @@ const EjectButton = (props: EjectButtonProps, context) => {
       color={'transparent'}
       className={classes([
         'Fabricator__PrintAmount',
-        amount * 2_000 > available && 'Fabricator__PrintAmount--disabled',
+        amount * 100 > available && 'Fabricator__PrintAmount--disabled',
```

Comment:
same here

Reply:
Why on SMALL_MATERIAL_AMOUNT? You can't eject 0.1 of sheet from what i know

Reply:
Not SMALL_MATERIAL_AMOUNT we have to use SHEET_MATERIAL_AMOUNT because 2000 was the original value of a  sheet